example/40-validatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: gardener-extension-validator-gcp
+webhooks:
+- name: validation.gcp.provider.extensions.gardener.cloud
+  rules:
+  - apiGroups:
+    - "core.gardener.cloud"
+    apiVersions:
+    - v1alpha1
+    - v1beta1
+    operations:
+    - CREATE
+    - UPDATE
+    resources:
+    - shoots
+  failurePolicy: Fail
+  objectSelector: {}
+  namespaceSelector: {}
+  admissionReviewVersions: ["v1", "v1beta1"]
+  sideEffects: None
+  clientConfig:
+    url: "https://localhost:9443/webhooks/validate"
+    caBundle: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUU0akNDQXNvQ0NRRHVLOTQ5QnF0eUt6QU5CZ2txaGtpRzl3MEJBUXNGQURBek1Sb3dHQVlEVlFRS0RCRkgKWVhKa1pXNWxjaUJGZUdGdGNHeGxjekVWTUJNR0ExVUVDd3dNVUhKdmRtbGtaWElnUVZkVE1CNFhEVEU1TVRFeQpNREV4TURRd00xb1hEVEk1TVRFeE56RXhNRFF3TTFvd016RWFNQmdHQTFVRUNnd1JSMkZ5WkdWdVpYSWdSWGhoCmJYQnNaWE14RlRBVEJnTlZCQXNNREZCeWIzWnBaR1Z5SUVGWFV6Q0NBaUl3RFFZSktvWklodmNOQVFFQkJRQUQKZ2dJUEFEQ0NBZ29DZ2dJQkFOV0UrY0F4MGg3b1pQems4UzdWY3lsbUx3YVZ3SU9BZkpkd0NSYkpuUmZNbm1RUQpqNWFaeExqN3VGR2ZhMmdxeDNFZkxZNU1YYXA1V21GSUVNM0Y4N1AxdmRzRU1lRW9ndUNLb0NsVk95cGZLQnZQCm10cXRDUWEzK3dwOUwvSlg3dGZubmJKQVZ3RTF0RzY4Mk5OajVwaVIxUXlrc1dXTGl6MFpMdG1MaFR1QlgxQWgKOW9vWnNTZy9FS1JkUmZvU1hGUWE3bFlmTkthVWNuRUNDSnNtQ0l2NjNJWFo0T1dHWU85aXNwRENkZ3ZPMFRCeApwZ3Q0NUZuUmQ5U1lhTE1kanY2WGNaL0hmYmFhWHk0QUZIeU5Lb1NaWVZxUDg0TVVia1BGQkhHZ1ZERENCa3VSCm9tOTVxb1BTK2Q0S0dCN1BnV1VnaXJLVWFpWHdIeWNNUFQ1d0Y3ZHpFWjdtUnNTakpTR1ZpK2NXZnRuMjBBNjAKanp4aG95UjZNb09nSFp6QTd2VC84THpPcDBhQXR6TFhIakJzZ1pJMlhYcU9oS01LL1BSRFE3WlRxdGo1WFpsZQo4c3hPMG51VFlaQllMcjFYTGkrYWU2ZmxtTXozeUZPMmZMR2R0dzRmMFI4Ky85aERZUEFhNHluRFBZWVdnbFVXCnJqWjFXZGptMG12bXAvaDVNeFJCbjlzTFVKb0tUVWV6Q0dzcDBXRkx4MG1BZHYxTjZIZVpUaDVxWG1XdWpjdWkKeGlXcUVNaVlWcGJhTmpWTm1OQWhUUDY4VWNkVEhRQUpnYlU5SVY4TmtmYklIcFRjY3FFaGFqUmFFdmo0SmlIbApnUzQ0V0d6VzQ0SjJKdHJzamZlYVRKZWtpbFh0WjNVdFAreXUxUjZ0NlJyQmc2bmNySmVySWRQc1RkZ0pBZ01CCkFBRXdEUVlKS29aSWh2Y05BUUVMQlFBRGdnSUJBSzRyYUVmMEI0dGJzakhiNmIxa0hUVnMyKyt5QjdXMmVxWTgKSUtpZE1VcEFSSjhSWmVZVEh5N0lFUHZYQUQ4aW9IOW1Pd1dPT2ZyRUdCN2pBUjJCb1IyR2ZSd3hoM3RWM0RLdgoydEZZeGVKWlNuTEFuZmNLQzFlZ3duT1JuOTRjRk54bkhOQkRjS2RiU3FHL2VSNFdpVWdjV1YwbmNqVHNTUXZjCnMrdkd1WlQwUFlLcmxHYlB4VkhXSkJEdXFCQ3djczhIWEk4NzlSS3JmMTZOWVZEay9vTE5wUXV1V2NDQ0pDQzMKaWxnMTFGbkR6L1ZBMkppMWdhbGtwZEVxMHVKemlmcnY4S0ZrdmF3TUlFRWdCMTYwQnFrNmdsdkt5dWVUSHFLYQpuaCtJV3IzYlY1VHQ4QXNPdFZCdWJxempHOU9VU2g1LzZYT0FyTXhTaXBwT2llYmNYcFhrZWMyTldMRGNnQjlrCjB1T1FrTzZvUjBxZlFobXAwTEpsaEhCVGNHZkJsK3Yzd1B2RE5KS1hTM2xMVWg3L0J3d0piR3M0Mzd4TU10M1cKaGdzNTJWU0k1RU5EU2lmelplcnJzTXlQYzhHcXJ0NFhkYVRBU3VYNTlja0VDaUxSemFBbS9QQVl2N0Ntd0dvYQpLc0tOMzdZQWgrVXVONVc4Q3BLMzhFbG9Uc0ZVRzl0WEhYWmU5UG4wRjlXQU9Sb3YzVmlHTVZhZERmMzlmRDdzCjAwd1JrTnFNQy9CVFBtcHBqWXRieDQ1M2M3TWZrOGNNa0pnMGdTZnh3d2h0QUtibEszeXNOK3JsbnBMUDRPR0UKZHY4WHpKMUIrZVdzT3ZUcDI0TkRWZnJvQjFWbklmaDVKdnpiV1JZdjlLL09zdHBmWHgzZWRFR1p3djgxTnNHTQoxd0JSVkpJUgotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```

Comment:
Please use the CA which was used to issue the certificate in `example/validator-gcp-certs/tls.crt`.
```
Error from server (InternalError): error when creating "/Users/./k8s/00-local-dev/gcp/90-shoot-gcp.yaml": Internal error occurred: failed calling webhook "validation.gcp.provider.extensions.gardener.cloud": Post https://localhost:9443/webhooks/validate?timeout=10s: x509: certificate signed by unknown authority
```